src/libaktualizr/http/httpclient_test.cc

```diff
@@ -80,6 +80,25 @@ TEST(PostTest, put_performed) {
   EXPECT_EQ(json["data"]["key"].asString(), "val");
 }
 
+TEST(HttpClient, user_agent) {
+  {
+    HttpClient http;
+
+    const auto resp = http.get(server + "/user_agent", HttpInterface::kNoLimit);
+    const auto app = resp.body.substr(0, resp.body.find('/'));
+    EXPECT_EQ(app, "Aktualizr");
```

Comment:
Could you provide a brief comment to make it clear that you're testing the expected default here?

Reply:
Good point!
Done.